### PR TITLE
Bump NN, Maps and Common dependency versions to `112.0.0`, `10.8.0-beta.1` and `23.0.0-beta.1` respectively

### PR DESCRIPTION
**Pull request by Guardiola31337**
### Description

- Bumps NN, Maps and Common dependency versions to `112.0.0`, [`10.8.0-beta.1`](https://github.com/mapbox/mapbox-maps-android/releases/tag/v10.8.0-beta.1) and `23.0.0-beta.1` respectively

### CODE REVIEW

File: instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt [hunk 1]
```diff
@@ -241,8 +241,8 @@ class RouteRefreshTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.ja
             alternativesMetadata.navigationRoute.id
         )
         assertNotEquals(alternativesMetadataLegacy, alternativesMetadata)
-        assertEquals(266.0, alternativesMetadataLegacy.infoFromStartOfPrimary.duration, 1.0)
-        assertEquals(275.0, alternativesMetadata.infoFromStartOfPrimary.duration, 1.0)
+        assertEquals(267.9, alternativesMetadataLegacy.infoFromStartOfPrimary.duration, 1.0)
+        assertEquals(277.0, alternativesMetadata.infoFromStartOfPrimary.duration, 1.0)
```
Comment (Guardiola31337) [inline instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt:245]: Noting that I had to change the values here. Is that expected? @RingerJK @LukasPaczos

Comment (Guardiola31337) [reply instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt:245]: Probably yes due to

> Fixed calculation of alternative route info (duration and distance)

cc @DenisPryt

Comment (LukasPaczos) [reply instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt:245]: In this case we should add it to our changelog as well.

Comment (Guardiola31337) [reply instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt:245]: Great point. Just added it 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ Mapbox welcomes participation and contributions from everyone.
...
 - Updated `NavigationView` to allow drawing of the info panel behind the translucent navigation bar. [#6145](https://github.com/mapbox/mapbox-navigation-android/pull/6145)
 - Optimized vanishing line updates (`MapboxRouteLineApi#updateTraveledRouteLine`) when going through restrictions and legs aren't styled independently. [#6169](https://github.com/mapbox/mapbox-navigation-android/pull/6169)
+- Added support for _Onboard_ `snapping_include_static_closures`. [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
+- Fixed a race condition that could cause a hang during `TileStore` clean up. [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
+- Fixed a crash when the `TileStore` service process is killed by the Android system. [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
+- Fixed an issue where some callbacks for completed operations would be invoked again if the `TileStore` service process was terminated. [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
+- Fixed calculation of alternative route info (duration and distance). [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
```
Comment (LukasPaczos) [inline CHANGELOG.md:14]:
```suggestion
- Fixed calculation of `AlternativeRouteMetadata` (duration and distance). [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,11 @@ Mapbox welcomes participation and contributions from everyone.
 #### Bug fixes and improvements
 - Updated `NavigationView` to allow drawing of the info panel behind the translucent navigation bar. [#6145](https://github.com/mapbox/mapbox-navigation-android/pull/6145)
 - Optimized vanishing line updates (`MapboxRouteLineApi#updateTraveledRouteLine`) when going through restrictions and legs aren't styled independently. [#6169](https://github.com/mapbox/mapbox-navigation-android/pull/6169)
+- Added support for _Onboard_ `snapping_include_static_closures`. [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
```
Comment (LukasPaczos) [inline CHANGELOG.md:10]:
```suggestion
- Added support for _Onboard_ `snapping_include_static_closures` route request parameter (`RouteOptions#snappingIncludeStaticClosures). [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
```

**Review by LukasPaczos — APPROVED** [review]
Just some changelog suggestion that can also be corrected separately during the release process.

**Review by RingerJK — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,11 @@ Mapbox welcomes participation and contributions from everyone.
 #### Bug fixes and improvements
 - Updated `NavigationView` to allow drawing of the info panel behind the translucent navigation bar. [#6145](https://github.com/mapbox/mapbox-navigation-android/pull/6145)
 - Optimized vanishing line updates (`MapboxRouteLineApi#updateTraveledRouteLine`) when going through restrictions and legs aren't styled independently. [#6169](https://github.com/mapbox/mapbox-navigation-android/pull/6169)
+- Added support for _Onboard_ `snapping_include_static_closures` route request parameter (`RouteOptions#snappingIncludeStaticClosures). [#6168](https://github.com/mapbox/mapbox-navigation-android/pull/6168)
```
Comment (VysotskiVadim) [inline CHANGELOG.md:10]: It could also be worth mentioning that map matching supports statically closed roads now

Comment (Guardiola31337) [reply CHANGELOG.md:10]: What do you mean? Isn't it the same as _Onboard_ (Offline) support?

**Pull request merged**